### PR TITLE
JW-928 - Use correct (linux like) path for config file

### PR DESCRIPTION
**Pull request by kaweue**

### CODE REVIEW

File: src/vnsw/agent/contrail/windows/msi/agent_msi.wxs [hunk 1]
```diff
@@ -5,6 +5,17 @@
         <Media Id="1" Cabinet="product.cab" EmbedCab="yes"/>
 
         <Directory Id="TARGETDIR" Name="SourceDir">
+            <Directory Id="CommonAppDataFolder">
+                <Directory Id="ROOTDIR" Name="ContrailVrouterAgent">
```
Comment (kaweue) [inline src/vnsw/agent/contrail/windows/msi/agent_msi.wxs]: This directory path is based on: https://docs.google.com/presentation/d/1s0S0yQloM7aFx4TxE2nidTz8gdQ5LWaD9GZXYMJM_pY/edit#slide=id.g25279196e7_1_145

Comment (m-kostrzewa) [reply src/vnsw/agent/contrail/windows/msi/agent_msi.wxs]: Now that I think about it, this dir layout it kinda doesn't make sense.
/etc, /var... are subdirs of ContrailVrouterAgent, but potentially other Contrail services (like NodeManager, DockerDriver) could also use /etc and /var. Should we have additional /etc, /var... under NodeManager/ and DockerDriver? I don't think so...

I think we should have something like Program Data/Contrail/{var},{etc}

What do you think?

Comment (kaweue) [reply src/vnsw/agent/contrail/windows/msi/agent_msi.wxs]: I think it is a good idea. I'll change those paths.

**Review by m-kostrzewa — APPROVED** [review]

File: src/vnsw/agent/contrail/windows/msi/agent_msi.wxs [hunk 2]
```diff
@@ -5,6 +5,17 @@
         <Media Id="1" Cabinet="product.cab" EmbedCab="yes"/>
 
         <Directory Id="TARGETDIR" Name="SourceDir">
+            <Directory Id="CommonAppDataFolder">
+                <Directory Id="ROOTDIR" Name="Contrail">
+                    <Directory Id="ETCDIR" Name="etc">
+                        <Directory Id="CONFIGDIR" Name="contrail">
+                            <Component Id="ContrailVrouterAgentConfig" Guid="3386587d-a358-4193-90e7-ad2f659113e5" Win64="yes">
```
Comment (katrybacka) [inline src/vnsw/agent/contrail/windows/msi/agent_msi.wxs]: Please use uppercase

**Review by katrybacka — CHANGES_REQUESTED** [review]

**Review by katrybacka — APPROVED** [review]

**Pull request merged**